tests/test_libmount.py

```diff
@@ -1,17 +1,23 @@
 from concurrent.futures import ThreadPoolExecutor
 from contextlib import contextmanager, suppress
 from pathlib import Path
+from shutil import rmtree
 from unittest import TestCase
 from uuid import uuid4
 
 from snekbox import libmount
 
 
 class LibMountTests(TestCase):
-    def setUp(self):
-        self.temp_dir = Path("/tmp/snekbox-test")
-        self.temp_dir.mkdir(exist_ok=True, parents=True)
-        super().setUp()
+    temp_dir = Path("/tmp/snekbox")
+
+    @classmethod
+    def setUpClass(cls):
+        cls.temp_dir.mkdir(exist_ok=True, parents=True)
+
+    @classmethod
+    def tearDownClass(cls):
+        rmtree(cls.temp_dir, ignore_errors=True)
```

Comment:
Actually speaking of `tempfile`, that's probably a better case, I've refactored this to use `tempfile.TemporaryDirectory` instead of manually doing this [62954cd](https://github.com/python-discord/snekbox/pull/159/commits/62954cd078cbcb4719cb1a1b726fe3672a17c495)